src/PageContent/Calendar/ContactContent.js

```diff
@@ -0,0 +1,184 @@
+                    phone: this.props.delivery.donatingAgency.contact.phone,
+                    memberList: this.props.delivery.donatingAgency.contact
+                        .memberList
+                }
+            };
+        } else if (this.props.accountType === AccountType.RECEIVING_AGENCY) {
+            this.state = {
+                contact: {
+                    name: this.props.delivery.receivingAgency.contact.name,
+                    phone: this.props.delivery.receivingAgency.contact.phone
+                }
+            };
+        }
+
+        this.edit = this.edit.bind(this);
+        this.handleChange = this.handleChange.bind(this);
+        this.handleChangeDA = this.handleChangeDA.bind(this);
+    }
+    edit() {
+        this.setState({
+            edit: true
+        });
+    }
+
+    handleChange(e) {
+        e.preventDefault();
+        this.setState({
+            contact: {
+                name: e.target.name.value,
+                phone: e.target.phone.value,
+                edit: false
+            }
+        });
+    }
+
+    handleChangeDA(e) {
+        e.preventDefault();
+        let index = e.target.primaryContact.value;
+        let memberName = this.state.contact.memberList[index].name;
+        let memberPhone = this.state.contact.memberList[index].phone;
+        let memberList = this.state.contact.memberList;
+        this.setState({
+            contact: {
+                name: memberName,
+                phone: memberPhone,
+                memberList: memberList
+            },
+            edit: false
+        });
+    }
+
+    render() {
+        return (
+            <div className="wrapper">
+                <img className="content-icon" src={phone} alt="volunteer" />
+                <div className="content-wrapper content-wrapper-description">
+                    {this.props.accountType === AccountType.RECEIVING_AGENCY ? (
+                        <h1 className="section-header">
+                            Primary Contact for Delivery
+                        </h1>
+                    ) : this.props.accountType ===
+                    AccountType.DONATING_AGENCY_MEMBER ? (
+                            <h1 className="section-header">
+                            Primary Contact for Pick Up
+                            </h1>
+                        ) : null}
+                    {!this.state.edit ? (
+                        <div>
+                            {!this.state.name ? (
+                                <div className="content-details-wrapper">
+                                    <p className="content-details contact-content">
+                                        {this.state.contact.name} ({
+                                            this.state.contact.phone
+                                        })
+                                    </p>
+                                </div>
+                            ) : (
```

Comment:
the block above and below are now the same, there's no need for the `this.state.name ?` check anymore